.github/workflows/build-timestamped-master.yml

```
@@ -5,43 +5,14 @@ on:
     branches:
       - main
     paths-ignore:
+      - 'load-tests/**'
       - '*.md'
+      - 'docs/**'
   workflow_dispatch:
 
 jobs:
-  build:
-    runs-on: ubuntu-latest
-    if: github.repository_owner == 'ballerina-platform'
-    steps:
-      - name: Checkout Repository
-        uses: actions/checkout@v3
-      - name: Set up JDK 17
-        uses: actions/setup-java@v3
-        with:
-          distribution: 'temurin'
-          java-version: 17.0.7
-      - name: Change to Timestamped Version
-        run: |
-          startTime=$(TZ="Asia/Kolkata" date +'%Y%m%d-%H%M00')
-          latestCommit=$(git log -n 1 --pretty=format:"%h")
-          VERSION=$((grep -w 'version' | cut -d= -f2) < gradle.properties | rev | cut --complement -d- -f1 | rev)
-          updatedVersion=$VERSION-$startTime-$latestCommit
-          echo $updatedVersion
-          sed -i "s/version=\(.*\)/version=$updatedVersion/g" gradle.properties
-      - name: Grant execute permission for gradlew
-        run: chmod +x gradlew
-      - name: Build with Gradle
-        env:
-          packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
-          packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
-          publishUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
-          publishPAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
-        run: |
-          ./gradlew publish --scan --no-daemon
-      - name: Generate CodeCov Report
-        uses: codecov/codecov-action@v2
-      - name: Upload Artifact
-        uses: actions/upload-artifact@v2
-        with:
-          name: ballerina-runtime
-          path: target/ballerina-runtime/
+  call_workflow:
+    name: Run Build Workflow
+    if: ${{ github.repository_owner == 'ballerina-platform' }}
+    uses: ballerina-platform/ballerina-standard-library/.github/workflows/build-timestamp-master-template.yml@main
+    secrets: inherit
```

```suggestion
    secrets: inherit

```
EoF new line is the standard. Check in other files

---

Sure, Changed with [a954788](https://github.com/ballerina-platform/module-ballerina-edi/pull/31/commits/a954788d638ec20c4dd61686bf24250484dfc80f).